src/5e-SRD-Suggested-Characteristics.json

```diff
@@ -0,0 +1,204 @@
+[
```

Comment:
Since it's a 1:1 relationship, is there any reason we shouldn't just combine Suggested Characteristics with Backgrounds?